src/main/view/screens/ScreensEnum.cpp

```diff
@@ -10,6 +10,7 @@ const char* ScreensEnumText[] = {
 	"MENU",
 	"OPTIONS",
 	"PAUSED",
+	"GAMEOVER",
 0};
```

Comment:
I wrote this. And I remember getting an error without it, but I just yanked it out and tested and it compiled and ran without it, so maybe we can ditch it.